modules/oadp-creating-backup-cr.adoc

```diff
@@ -22,7 +22,9 @@ You back up Kubernetes images, internal images, and persistent volumes (PVs) by
 
 .Procedure
 
-. Retrieve the `backupStorageLocations` CRs:
+. Retrieve the `backupStorageLocations` CRs by entering the following command:
+
+:
```

Comment:
There is an errant `:` here that is throwing off the rendering of the whole procedure. Can you remove this and the blank line above it?

![image](https://user-images.githubusercontent.com/7443887/187764336-64a6977d-7e67-4ba6-b703-3324c6c31feb.png)